pkg/features/kube_features.go

```diff
@@ -364,6 +370,7 @@ var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureS
 	DynamicProvisioningScheduling:               {Default: false, PreRelease: utilfeature.Alpha},
 	VolumeSubpathEnvExpansion:                   {Default: false, PreRelease: utilfeature.Alpha},
 	KubeletPluginsWatcher:                       {Default: false, PreRelease: utilfeature.Alpha},
+	VolumeSnapshot:                              {Default: true, PreRelease: utilfeature.Alpha},
```

Comment:
Normally we don't enable alpha feature by default until it is beta.

Reply:
Got it.  Will fix it.